pyproject.toml

```diff
@@ -1,7 +1,55 @@
 [build-system]
-
 requires = ["setuptools",
             "setuptools_scm",
             "wheel"]
-
 build-backend = 'setuptools.build_meta'
+
+[ tool.gilesbot ]
```

Comment:
Decide if we want this or just add to a changelog file manually?

Reply:
Lets leave this in, for the first release I don't think we need a changelog but for future releases this is 👍